base/abstractarraymath.jl

```diff
@@ -48,9 +48,11 @@ _sub(t::Tuple, s::Tuple) = _sub(tail(t), tail(s))
 """
     dropdims(A; dims)
 
-Remove the dimensions specified by `dims` from array `A`.
-Elements of `dims` must be unique and within the range `1:ndims(A)`.
-`size(A,i)` must equal 1 for all `i` in `dims`.
+Return an array with the same data as `A`, but with dimensions—specified by
+`dims`—removed. The two arrays share the same underlying data, so that the
```

Comment:
```suggestion
Return an array with the same data as `A`, but with the dimensions specified by
`dims` removed. The two arrays may share the same underlying data, such that the
```

Reply:
Is it not guaranteed? `dropdims` uses `reshape`, and the docs for `reshape` seem to suggest it is always the case.

Reply:
Oh, I don't know. I just assumed it wasn't because it wasn't previously documented, and didn't want to constrain implementers. If this is in the interface, we should remove the "may".